src/DDTrace/Processing/TraceAnalyticsProcessor.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace DDTrace\Processing;
```

Comment:
I'm curious if we should consider naming this `namespace DDTrace\Filters` to be in line with other tracers? Maybe filter functionality is something we do in a separate PR and refactor `TraceAnalyticsProcessor` into a filter. Maybe we can add an `@TODO` comment? Or we could just mention it in the backlog. :)

Reply:
This is a good discussion! If you don't mind I'd prefer to add it into the comment because I feel like `xFilter` would be a sort of misleading at this stage. There is no mechanism to filter out spans. Of course this can be translated to filter once we put this mechanism in place. Being it final it should be no risky at all. Would you agree?